tests/test_bills.py

```diff
@@ -125,12 +124,11 @@ def test_last_action_date_has_already_occurred(bill, event):
         item = some_event.agenda.create(order=1)
         item.related_entities.create(bill=some_bill)
 
-
     # Assert the bill occurs on both agendas.
     assert Event.objects.filter(agenda__related_entities__bill=some_bill)\
                         .count() == 2
 
-    last_action_date = some_bill.get_last_action_date()
+    last_action_date = some_bill.councilmatic_bill.get_last_action_date()
 
     # Assert the last action matches the event that has already occurred.
-    assert last_action_date == two_weeks_ago
+    assert last_action_date.date() == two_weeks_ago.date()
```

Comment:
I kept this test here because it uses so much of the Metro-specific test infrastructure. It would be ideal to move it upstream, but given our limitations on hours, I thought this would suffice for now.